app/src/lib/markdown-filters/node-filter.ts

```diff
@@ -104,15 +106,24 @@ export const buildCustomMarkDownNodeFilterPipe = memoizeOne(
  */
 export async function applyNodeFilters(
   nodeFilters: ReadonlyArray<INodeFilter>,
-  parsedMarkdown: string
-): Promise<string> {
-  const mdDoc = new DOMParser().parseFromString(parsedMarkdown, 'text/html')
+  markdownEmitter: MarkdownEmitter
+): Promise<void> {
+  if (markdownEmitter.latestMarkdown === null || markdownEmitter.isDisposed) {
+    return
+  }
+
+  const mdDoc = new DOMParser().parseFromString(
+    markdownEmitter.latestMarkdown,
+    'text/html'
+  )
 
   for (const nodeFilter of nodeFilters) {
     await applyNodeFilter(nodeFilter, mdDoc)
+    if (markdownEmitter.isDisposed) {
+      break
+    }
```

Comment:
@tidy-dev  I just tested this by adding a `await sleep(5000)` right above the `if` and setting a breakpoint in the `break` and it didn't stop there. From the console I checked, and it seems the emitter has a `disposed` property, but not `isDisposed`.

From the looks of it, the API changed so I bumped `@types/event-kit` and updated all the required references (including this `disposed`) in eba06ef67a0779a08f087dc0e265bedede92604a and now all works as expected 🎉 

Feel free to revert/remove/change anything from that commit if needed!

Reply:
Looks good to me.